src/test/java/org/jenkinsci/plugins/workflow/pipelinegraphanalysis/StatusAndTimingTest.java

```diff
@@ -430,15 +431,18 @@ public void inputTest() throws Exception {
     @Test
     public void busyStepTest() throws Exception {
         WorkflowJob job = j.jenkins.createProject(WorkflowJob.class, "InputJob");
+        String sleep = "sh 'sleep 10000'\n";
+        if(SystemUtils.IS_OS_WINDOWS){
```

Comment:
There's actual pipeline syntax for this, btw -- this will work I think, but might be more natural to use that.